docs/zkapps/index.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-title: zkApp Overview
+title: zkApps Overview
 sidebar_label: Overview
 hide_title: true
-description: Get an understanding of zkApps, the smart contracts of Mina Protocol. zkApps are powered by zero knowledge proofs and off-chain execution. A quickstart guide to deploy zkApps and resources for learning TypeScript.
+description: zkApps build the smart contracts of Mina Protocol. zkApps are powered by zero knowledge proofs and off-chain execution. A quickstart guide to deploy zkApps and resources for learning TypeScript.
```

Comment:
The first sentence of the description may be a little confusing to some. zkApps are built with smart contracts, but I don't think it makes sense to say zkApps build the smart contracts.